src/components/home/Home.js

```diff
@@ -12,38 +14,46 @@ import Car from './Car';
 const modelsUrl = ' https://carhubackend.herokuapp.com/models_b';
 
 const Home = () => {
-  let stateModel = [];
-  stateModel = useSelector((state) => state.myModels);
+  const stateModel = useSelector((state) => state.myModels);
   const dispatch = useDispatch();
 
   const navToggle = () => {
     const homeLink = document.querySelector('#home_link');
     const createLink = document.querySelector('#create_link');
     const reserveLink = document.querySelector('#reserve_link');
     const logoutLink = document.querySelector('#logout_link');
+    const deleteLink = document.querySelector('#delete_link');
 
+    deleteLink.classList.remove('selected_nav');
     logoutLink.classList.remove('selected_nav');
     createLink.classList.remove('selected_nav');
     homeLink.classList.add('selected_nav');
     reserveLink.classList.remove('selected_nav');
   };
 
   const getModels = () => {
-    fetch(modelsUrl, {
+    axios.get(modelsUrl, {
       headers: {
-        Authentication: localStorage.getItem('token'),
+        authorization: localStorage.getItem('token'),
       },
-    }).then((data) => {
-      data.json().then((dataJson) => dispatch(models(dataJson)));
-    });
+    })
+      .then((response) => response)
+      .then((response) => {
+        const { data } = response;
+        console.log(data);
```

Comment:
Thanks for your work. I just suggest if you can try removing the "console.logs" in the code to make it look more professional

Reply:
Sure, i am on it